cmd/metrics.go

```diff
@@ -72,6 +72,27 @@ func (c *minioCollector) Collect(ch chan<- prometheus.Metric) {
 	}
 	s := objLayer.StorageInfo(context.Background())
 
+	cacheObjLayer := newCacheObjectsFn()
+	if cacheObjLayer != nil {
+		cs := cacheObjLayer.StorageInfo(context.Background())
+		ch <- prometheus.MustNewConstMetric(
+			prometheus.NewDesc(
+				prometheus.BuildFQName("minio", "disk", "storage_bytes"),
+				"Total cached storage available for current Minio server instance",
```

Comment:
Using "Total cached storage available..." and "Total free cached storage available..." is confusing. Instead, might want to consider  "Total cache capacity for current Minio server instance"

Reply:
Will change it to "Total cache capacity on current Minio server instance"